src/checks/lowercase_key.rs

```diff
@@ -0,0 +1,51 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct LowercaseKeyChecker {}
+
+impl Check for LowercaseKeyChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        let line_str: Vec<&str> = line.raw_string.split('=').collect();
+        let key = line_str[0];
+        if key.to_uppercase() == key {
```

Comment:
The method `to_uppercase()` returns `String`, but the variable `key` is `&str`.
I think you should convert `&str` to `String`.
```suggestion
        if key.to_uppercase() == key.to_string() {
```

Reply:
thx, i think it will be better
```rust
let key = line_str[0].to_string();
if key.to_uppercase() == key {
```